torch/xpu/__init__.py

```diff
@@ -398,6 +398,11 @@ def empty_cache() -> None:
         torch._C._xpu_emptyCache()
 
 
+def _host_allocator():
```

Comment:
Why do you need this one on the python side?

Reply:
To align with CUDA's API [_host_allocator](https://github.com/pytorch/pytorch/blob/eeb626b46a4d606d6900aec8c690feb8a9d6e88b/torch/cuda/memory.py#L54).
Remove it for now if you have any concerns. It is OK to add back if necessary.

Reply:
Yeah lets not add it unless we have a use for it!